File: onclick/onclick.py
import random
def display(total):
    print("\t\t\t\t\t",total)
print("1)increase")
print("2)decrease")
print("3)reset")
print("4)random")
print("5)exit")
total =0
choose =0
while choose!=5 :
    choose=int(input("Enter you choose:"))
    if choose == 1:
        total+=1
        display(total)
    elif choose == 2:
        total-=1
        display(total)
    elif choose == 3:
        total =0
        display(total)
    elif choose ==4:
        total= random.randrange(0,100)
        display(total)
    else:
        print("please enter the correct choose")
        choose=int(input("Enter you choose:"))
    print("1)increase \t 2)decrease \t 3)reset \t 4)random \t 5)exit")

        

File: onclick/onclick.js
const increase = document.getElementById("increase");
const reset = document.getElementById("reset");
const decrease = document.getElementById("decrease");
const random = document.getElementById("random");
const number = document.getElementById("head");
let count = 0;

increase.onclick = function() {
    count++;
    number.textContent = count;
}
decrease.onclick = function() {
    count--;
    number.textContent = count;
}
reset.onclick = function() {
    count = 0;
    number.textContent = count;
}
random.onclick = function() {
    count = Math.floor(Math.random() * 100) + 1;
    number.textContent = count;
}